templates/boltzmann/middleware/jwt.ts

```diff
@@ -32,7 +32,7 @@ function authenticateJWT ({
     throw new Error(
       `To authenticate JWTs you must pass the path to a public key file in either
 the environment variable "AUTHENTICATION_KEY" or the publicKey config field
-https://www.boltzmann.dev/en/docs/{{ version }}/reference/middleware/#authenticatejwt
+https://www.boltzmann.dev/en/docs/v{{ version }}/reference/middleware/#authenticatejwt
```

Comment:
This was a change made by Jeff in #95. It looked like it might not apply cleanly today, so I cherry-picked the commit and put it in here. I think it's low risk to do so, and it'll get his fix in.